api/v1/views/amenities.py

```diff
@@ -0,0 +1,73 @@
+from api.v1.views import app_views
+from models import storage
+from models.amenity import Amenity
+from flask import jsonify, abort, request
+
+@app_views.route('/amenities', method=['GET'], strict_slashes=False)
+def get_amenities():
+    amenities = storage.all(Amenity).values
```

Comment:
add () to 'values'

Reply:
i have add parenthese after the function value